package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "city-in-the-south",
+  "version": "1.0.0",
+  "description": "## PROBLEM:",
+  "main": "index.js",
+  "scripts": {
+    "server": "fuser -k 5000/tcp;clear; nodemon server/index.js",
+    "client": "npm start --prefix client",
+    "kill": "fuser -k 5000/tcp; fuser -k 3000/tcp;",
+    "dev": "npm run kill;killall node;concurrently \"npm run client\" \"npm run server\"",
```

Comment:
```suggestion
    "start:dev": "npm run kill;killall node;concurrently \"npm run client\" \"npm run server\"",
```

Reply:
I think it is ok, no need for extra word!